framework/src/modules/dpos_v2/commands/vote_delegate.ts

```diff
@@ -184,18 +238,57 @@ export class VoteDelegateCommand extends BaseCommand {
 				// Sort account.unlocking
 				sortUnlocking(voterData.pendingUnlocks);
 
-				if (voterData.pendingUnlocks.length > MAX_UNLOCKING) {
-					throw new Error(`Pending unlocks cannot exceed ${MAX_UNLOCKING.toString()}.`);
+				if (voterData.pendingUnlocks.length > MAX_NUMBER_PENDING_UNLOCKS) {
+					this.events.get(DelegateVotedEvent).error(
```

Comment:
Comment about line 235: Why is it `height+1` and not just `height`?

Reply:
That's probably from the past, but i dont remember why it is like this 🤔

Reply:
Would be nice to figure this out..

Reply:
I think it's a bug, I will create an issue

Reply:
https://github.com/LiskHQ/lisk-sdk/issues/7792